sm/plugin_sm/src/plugin_manager.rs

```diff
@@ -73,15 +74,19 @@ impl Plugins for ExternalPlugins {
     }
 }
 
+const PLUGIN_CMD: &str = "list";
```

Comment:
I believe we already have this constant defined in the plugin.rs, most likely it only needs to be made pub